package.json

```diff
@@ -19,6 +19,7 @@
     "@sentry/webpack-plugin": "3.2.1",
     "@unleash/proxy-client-react": "4.5.2",
     "classnames": "2.5.1",
+    "jwt-decode": "^4.0.0",
```

Comment:
New dependency! Can you please remove the `^`? We rely on Dependabot to handle everything.

Also, I'd be curious to know about the decision to use jwt-decode. What was your selection criteria?

Reply:
I'm re-reading my comment and I realized it is a bit curt. :sweat_smile: 

`jwt-decode` is probably fine but it would be nice to know why we chose it, especially in the commit message for the future - is it the most popular or well maintained library for this? Did we look into any alternatives, etc...? I don't have any specific problem with `jwt-decode` itself I'm just totally ignorant so some extra context would be nice. :smile:

Reply:
Alright, that makes sense. I removed the `^`, and added information into the commit, hopefully that's what you meant. :)